crates/indexer/src/models/signatures.rs

```diff
@@ -315,11 +319,36 @@ impl Signature {
             transaction_block_height,
             signer,
             is_sender_primary,
-            type_: String::from("secp256k1_ecdsa_signature"),
-            public_key: s.public_key.to_string(),
+            type_: String::from("single_key_signature"),
+            public_key: "Not implemented".into(),
```

Comment:
Why "not implemented"?

Reply:
legacy indexer, should have left a comment